### DISCUSSION TITLE
Allow only certain users to login

### DISCUSSION THREAD

**Discussion by emmarvpol** [nextauthjs/next-auth#2253] [Help] [answered]
Hello!
Im working on a project in which I want to add login functionality. Next-auth is good enough but I can't find how to allow only certain users to login. Is there any way? 

Thanks!

**Comment by balazsorban44** [comment, marked as answer]
https://next-auth.js.org/configuration/callbacks#sign-in-callback you can use the signIn callback to control access. return true if a user is allowed to sign in, or return false if they are not. optionally you can return a relative path (string) where the unauthorized users will be redirected to

**Reply by emmarvpol** [reply]
Thanks you very much!!
